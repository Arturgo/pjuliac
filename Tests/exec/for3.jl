
# la variable de la boucle for est locale

function f(x)
    print(x, ".")
    x
end

n = 42
println(n)

for n = f(1) : f(10)
    print(n)
end
println()

println(n)

