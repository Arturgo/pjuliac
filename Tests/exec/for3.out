42
1.10.12345678910
42
